### ISSUE TITLE
Jetpack connect notice is missing features description for non-US stores

### ISSUE THREAD

**Issue by juliaamosova** [Automattic/woocommerce-services#2022]
There is a back story of this issue that can be seen here: https://github.com/woocommerce/woocommerce-admin/issues/4276. 

**Tl;dr:** If the user has a store that is not based in the US and that if they have WC Services installed and activated but Jetpack is only installed but not activated, the Jetpack connect notice is missing the names of the features that merchant will get if Jetpack is connected. 

So this particular issue in this repo is related to updating the copy of the notice in particular.  

**To reproduce:**

1. Either on existing site or brand new site, first, set the store address to the US via `WooCommerce / Settings / General`.
2. Install Jetpack but don't activate it.
3. Install WC Services and activate it. 
4. You should have a Jetpack connect notice appear letting you know about the features you'd get if connect to Jetpack. It looks good for US stores - verify it is as follows:

![j1](https://user-images.githubusercontent.com/19143190/81120163-1eec6b00-8efa-11ea-95b8-ce38e50c03d6.jpg)

5. Click on `Activate Jetpack and connect` and confirm that the Jetpack connection completion notice is being displayed and is specifying what features the user now can enjoy:

![j2](https://user-images.githubusercontent.com/19143190/81120230-417e8400-8efa-11ea-96bd-a1cd96f8004d.jpg)

6. Next, deactivate Jetpack (but don't delete it).
7. Change store address to any other than the US. For example, the UK. Observe the Jetpack connect notice is missing the names of the features the user gets access to if connect to Jetpack:

![j3](https://user-images.githubusercontent.com/19143190/81120309-71c62280-8efa-11ea-83c2-d35571a2c057.jpg)

8. Click on `Activate Jetpack and connect` - observe that the Jetpack connection completion notice is missing the names of the features the user can now enjoy:

![j4](https://user-images.githubusercontent.com/19143190/81120361-8e625a80-8efa-11ea-8657-fff0571b8f4f.jpg)

<hr>

Note that updating the notice to match the notice that is shown to the US stores won't really solve the issue here because of WC Services features only available to the US stores.

**Comment by efixler** [comment]
So what should the copy here be for non-US stores?
